src/components/cells/DescriptionCell.vue

```diff
@@ -153,7 +144,7 @@ td {
 
 .description-cell .tooltip {
   background-color: $white;
-  border-radius: .5rem;
+  border-radius: 0.5rem;
```

Comment:
Note sure about this one either.

Reply:
This rule is forced by Prettier too. The zero will be removed by the build process anyway.